templates/components/products/related.html

```diff
@@ -5,7 +5,7 @@
 <ul class="js-product-list" {{#if settings.data_tag_enabled}} data-list-name="Related Products" {{/if}}>
     {{#each this}}
         <li>
-            {{>components/products/card settings=../settings theme_settings=../theme_settings event="list" position=(add @index 1)}}
+            {{>components/products/card-alternate settings=../settings theme_settings=../theme_settings event="list" position=(add @index 1)}}
```

Comment:
@mgbelegu this is nit-picking, but `card-alternative` looks better to my taste. Feel free to ignore this comment if you feel different 👍🏻.

Reply:
@marcinkrzeminski I use alternate, because that word is used by default in other bigcommerce areas thats why. Let's keep it this way if you don't mind :)